TrackletAlgorithm/TrackletEngineUnit.h

```diff
@@ -55,8 +55,18 @@ class TrackletEngineUnit {
 
     const ap_uint<1>* stubptinnertmp = getPTInnerLUT<Seed,iTC>();
     const ap_uint<1>* stubptoutertmp = getPTOuterLUT<Seed,iTC>();
-
-    if ( Seed <= TF::L5L6 ){
+    if ( Seed == TF::L3L4 ){//Split up Outer L3L4 table - reduced LUT usage for some reason 
+      for(unsigned int i=0;i<kNBitsPTLutInner;i++) {
+#pragma HLS unroll
+        stubptinnerlutnew_[i] = stubptinnertmp[i];
```

Comment:
@jasonfan393 Since this part is the same for all seeds ≤ L5L6, could you please rearrange the if statements to slightly reduce duplicate code? So something like this:
```c++
    if ( Seed <= TF::L5L6 ){
      for(unsigned int i=0;i<kNBitsPTLutInner;i++) {
#pragma HLS unroll
        stubptinnerlutnew_[i] = stubptinnertmp[i];
      }
      if ( Seed == TF::L3L4){//Split up Outer L3L4 table - reduced LUT usage for some reason
        for(unsigned int i=0;i<kNBitsPTLutOuter;i++) {
#pragma HLS unroll
          stubptouterlutnew1_[i] = stubptoutertmp[i];
          stubptouterlutnew2_[i] = stubptoutertmp[i+256];
        }
      }
      else {
        for(unsigned int i=0;i<kNBitsPTLutOuter;i++) {
#pragma HLS unroll
          stubptouterlutnew_[i] = stubptoutertmp[i];
        }
      }
    }
```